components/places/src/db/schema.rs

```diff
@@ -278,6 +285,12 @@ const CREATE_IDX_MOZ_HISTORYVISITS_VISITDATE: &str =
 const CREATE_IDX_MOZ_HISTORYVISITS_ISLOCAL: &str =
     "CREATE INDEX islocalindex ON moz_historyvisits(is_local)";
 
+const CREATE_IDX_MOZ_HISTORYTOMBSTONES_PLACE: &str =
```

Comment:
Did you find these indices made a difference? The [docs](https://sqlite.org/lang_createtable.html#uniqueconst) say "in most cases, UNIQUE and PRIMARY KEY constraints are implemented by creating a unique index in the database".

Reply:
Yeah, it's probably worth finding a way to measure this. We do searches based on this, and it fits the guidelines for not-bad indices, but so it seemed worth adding, but it's hard for me to measure.

Reply:
I "measured" this by looking at the query plan output, which showed me that you're right, and those indices aren't necessary if we do `PRIMARY KEY(place_id, visit_date)`! (ditto for UNIQUE, but PRIMARY KEY will let is omit the rowid)